rustls/src/webpki/client_verifier_builder.rs

```diff
@@ -195,13 +195,13 @@ mod tests {
     }
 
     #[test]
-    fn test_noauth() {
+    fn test_client_verifier_noauth() {
```

Comment:
Pre-existing issue, but maybe split `noauth` into `no_auth` while you're in here?

Reply:
Done.